src/nft/components/explore/Banner.tsx

```diff
@@ -33,13 +34,14 @@ const Banner = () => {
 
   useEffect(() => {
     /* Rotate through Top 5 Collections on 15 second interval */
+    let stale = false
+    if (hovered || stale) return
```

Comment:
Ideally the guard statement is the first thing in the function (before declaring `stale`), to make it easier to read. You should move `let stale = false` back above the `return`, and move the guard after the comment.

Reply:
then you get an eslint no-use-before-define error